src/qibolab/platforms/platform.py

```diff
@@ -33,7 +33,7 @@ def disconnect(self):
     def execute_pulse_sequence(self, sequence, nshots=1024, relaxation_time=None):
         if relaxation_time is None:
             relaxation_time = self.relaxation_time
-        return self.design.play(self.qubits, sequence, nshots=nshots, relaxation_time=relaxation_time)
+        return self.design.play(self.qubits, self.couplers, sequence, nshots=nshots, relaxation_time=relaxation_time)
```

Comment:
@scarrazza @AleCandido I would like your opinion about this point before we make an interface change.

The IQM 5-qubit chip that we are planning to control with Zurich Instruments uses coupler qubits to do two-qubit gates. These cannot be used for computation as they do not have readout lines. They only have flux line and are used to turn the interaction between the neighboring real qubits on when we are doing a 2q-gate and off otherwise.

Even though they do not have readout, I believe we should assign `Qubit` objects to them as we will probably need to calibrate the flux pulses we send to them to do 2q-gates. Most likely we will need different routines for this in qibocal too, as it is a different architecture than qw5q_gold.

The decision needed here is the following:
1. Hold and pass around the couplers using a new dictionary `self.couplers` as it is done here, or
2. include them in the existing `self.qubits` dictionary and add an attribute `role` in the `Qubit` object which specifies what the qubit is (eg. real, coupler, witness, etc.). This could also be useful for qw5q which has witness qubits (which we are not using but sometimes interfere with tests because they exist in the runcard).

Option 1 provides a cleaner distinction, however it cannot accomodate other qubit roles (such as the witness or something that may come up later) and also it may not be ideal to pass around the `self.couplers` argument if it is not relevant for most platforms.

Reply:
I perfectly agree with @scarrazza: there is always a trade-off between over-complicating existing objects and introducing too many new objects.
But I believe in this case you can really adopt the second approach, I don't believe that a `role` would complicate the `Qubit` object too much.

I didn't understand at the beginning that you might have qubits that are not used for computations. So, when we say "5-qubit chip" are there 5 qubits available for computations? or are there 5 physical qubits? (5 is just an example, of course)